wiki/People/The_Team/Nomination_Assessment_Team/Evaluations/en.md

```diff
@@ -0,0 +1,84 @@
+
+
+## Special Evaluations
+
+### Activity
+
+Beatmap Nominator activity is checked automatically by the NAT website on the 1st of every month. If it has found there are nominators not meeting the activity requirements, they will be flagged and posted in their respective mode's NAT channel for NAT members to review by the bnsite Discord bot. Review will include whether the BN has posted an absence notice recently, and double check to ensure the website check is accurate to the BN's activity status. 
+
+If it is found the BN is indeed not meeting activity, they will be either warned or removed depending on severity. BNs who were removed for activity in this way may apply again in 30 days, provided they have at least 8 mods to show they have come back to being active as a modder again.
+
+### Resignations
+
+Beatmap Nominators can resign from the group at any time via the NAT website. When this happens, it is given a 24 hour deadline and announced to the NAT in their respective Discord channel. Three NAT members will be assigned to evaluate the nominator one last time, in order to determine if the resignation is on Good terms or on Standard terms. BN who resign on good terms may be able to fast-track back to being a full nominator if they choose to apply again within 1 year of resignation. Those who resign on standard terms will have to start from probation if they pass their re-application. 
+
+Generally, good and standard terms evaluations are relatively simple. BNs who have not been recently warned or placed on probation, and were not about to be warned for issues will usually resign on good terms. Standard terms is typically used for BNs who have been warned recently, are on probation, or were about to be warned for issues on their upcoming current BN evaluation. 
+
+Once the good or standard terms consensus is set, the NAT will send the BN a message via the [mappersguild bot](https://osu.ppy.sh/users/23648635) informing them of their status and when they can reapply to BN if they so choose.
+
+### Mock Evaluations
+
+In addition to NAT members evaluating, mock evaluations may also be given out to BN members for new BN application evaluations. In mock evaluations, BNs will submit individual evaluations alongside the NAT, but their evaluation will not determine the final consensus for the application unless the NAT evaluators can not come to their own consensus. BNs will not be able to see the evaluation in group phase, but will be able to read the evaluations in an anonymized format once the application is archived. Mock evaluations allow the NAT to scout for future NAT members, as well as for BNs to see how NAT evaluate applications and learn from them if joining NAT is something they are interested in in the future.
```

Comment:
are all nominators able to see the archived evaluations in general or is this specific to the ones partaking in the mock evaluation? (this implies the former)

```suggestion
In addition to BN evaluations by NAT members, mock evaluations may also be given out to Beatmap Nominators for new BN application evaluations. In mock evaluations, nominators will submit individual evaluations alongside the NAT, but it will not determine the final result for the application unless the NAT evaluators cannot come to their own consensus. Nominators will not be able to see the evaluation in the group phase, but will be able to read the evaluations in an anonymised format once the application is archived. Mock evaluations allow the NAT to scout for future NAT members, as well as for Beatmap Nominators to see how the NAT evaluates applications and learn from them if joining the NAT in the future is something they are interested in.
```

Reply:
Only the application evaluations they participated in, not sure how to make this clearer

Reply:
Maybe like this

```suggestion
In addition to BN evaluations by NAT members, mock evaluations may also be given out to Beatmap Nominators for new BN application evaluations. In mock evaluations, nominators will submit individual evaluations alongside the NAT, but it will not determine the final result for the application unless the NAT evaluators cannot come to their own consensus. For the mock evaluation they participated in, the nominator will be able to read the individual evaluations in an anonymised format once the application is archived.

Mock evaluations allow the NAT to scout for future NAT members, as well as for Beatmap Nominators to see how the NAT evaluates applications and learn from them if joining the NAT in the future is something they are interested in.
```